lib/fun_with_flags/store/persistent/in_memory.ex

```diff
@@ -0,0 +1,163 @@
+defmodule FunWithFlags.Store.Persistent.InMemory do
+  alias FunWithFlags.{Config, Gate, Flag, Store.Persistent.Ecto.Record}
+
```

Comment:
It would be good to reference the Behaviour that was [released in v1.4.0](https://github.com/tompave/fun_with_flags/blob/master/CHANGELOG.md#v140):

```elixir
@behaviour FunWithFlags.Store.Persistent
```

Reply:
good to know :ok_hand: